janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLConfigOptions.java

```diff
@@ -65,6 +65,11 @@
                 "and always use LOCAL_QUORUM instead",
             ConfigOption.Type.MASKABLE, false);
 
+    public static final ConfigOption<Boolean> EXTERNAL_LOCKING =
+            new ConfigOption<Boolean>(CQL_NS, "external-locking",
+                    "True to use external locking mechanism instead mechanism provided by JanusGraph",
+                    ConfigOption.Type.MASKABLE, false);
```

Comment:
remove 8 spaces of indentation on lines 70 and 71

Reply:
Done